src/memphis.ts

```diff
@@ -23,8 +23,10 @@ import { buildSchema as buildGraphQlSchema, GraphQLSchema } from 'graphql';
 import * as broker from 'nats';
 import * as protobuf from 'protobufjs';
 
-import { Consumer, MsgHeaders, Producer, Station } from '.';
-import { MemphisError, generateNameSuffix } from './utils';
+import {
+  Consumer, MemphisConsumerOption, MsgHeaders, Producer, Station
```

Comment:
I couldn't find MemphisConsumerOption in index.ts

Reply:
It is in interfaces.ts file inside the nest folder

Reply:
but how is it imported? you are making import from "." which is "index.ts" and it is missing inside this file

Reply:
MemphisConsumerOption is exported from nest folder and index.ts exports the nest folder. So, any one importing "." has access to the interface. However, I've updated the imports in memphis.ts file to make it clear.